hack/update-bazel.sh

```diff
@@ -43,9 +42,11 @@ if ! grep -q "# gazelle:importmap_prefix" "${KUBE_ROOT}/vendor/BUILD"; then
   echo "# gazelle:importmap_prefix k8s.io/kubernetes/vendor" >> "${KUBE_ROOT}/vendor/BUILD"
 fi
 
-gazelle fix \
-    -build_file_name=BUILD,BUILD.bazel \
-    -external=vendored \
-    -mode=fix
+pushd "$(kube::realpath ${KUBE_ROOT})" > /dev/null 2>&1
+  gazelle fix \
```

Comment:
the alternate fix (mentioned by @jayconrod on slack) is to do
```bash
gazelle fix ... -repo_root="${KUBE_ROOT}" "${KUBE_ROOT}"
```
which is also clunky.

Reply:
it's clunky, but local tests of it seem to work.. and it's less clunky than subshells IMO